handbook/sales/records.md

```diff
@@ -1,6 +1,6 @@
 # How to record emails, calls, and notes
 
-The [source of truth for our communications with customers is HubSpot](../communication.md). 
+The [source of truth for our communications with customers is Salesforce](../communication.md). 
 
-- All emails interacting with a customer must be logged to HubSpot (either by using the HubSpot Chrome extension to add a "Log" checkbox in Gmail, or by BCC'ing our HubSpot account's private ingestion email address (see the "HubSpot BCC email address" note in 1password).
-- Notes from all phone calls with customers should be logged in HubSpot. The call notes should be logged to the Deal, rather than the Contact or Customer (i.e., `/deal/` should appear in the URL). Click on the **Calls** tab at the top of the deal overview, and click the **Log Call** button.
+- All emails interacting with a customer must be logged to Salesforce (either by using the [Einstein Activity Capture](https://help.salesforce.com/articleView?id=einstein_sales_aac.htm&type=5) or manually to the relevant object.
```

Comment:
```suggestion
- All emails interacting with a customer must be logged to Salesforce (either by using [Einstein Activity Capture](https://help.salesforce.com/articleView?id=einstein_sales_aac.htm&type=5 or by manually adding the email to the relevant object).
```

Reply:
This link isn't very helpful... I was hoping to go install it! :) What do I as a user need to do to get this?